src/core/evaluator.js

```diff
@@ -2167,6 +2167,7 @@ class PartialEvaluator {
     includeMarkedContent = false,
     sink,
     seenStyles = new Set(),
+    cropBox,
```

Comment:
Are you sure that this is the *correct* thing to use in general, since the thing that determines what we actually render is the `view` getter on the `Page`-instances (if nothing else it feels surprising/inconsistent)?